CMakeLists.txt

```diff
@@ -40,8 +40,9 @@ set(_itk_build_testing ${BUILD_TESTING})
 set(_itk_build_shared ${BUILD_SHARED_LIBS})
 set(BUILD_SHARED_LIBS OFF)
 set(elastix_GIT_REPOSITORY "https://github.com/SuperElastix/elastix.git")
-# Tag of commit "COMP: Quick-fix ITKElastix error: 'itkCostFunction.h' file not found"
+set(elastix_GIT_REPOSITORY "https://github.com/thewtex/elastix.git")
```

Comment:
I guess that second `set` call has effect (and keep the first one for easy reverting later), but this looks suspiciously like a rebase gone wrong.

Reply:
Since this has not been merged upstream, yet, we have to use the repository where it exists.